Increase timeout from 30 seconds to 360 seconds

Also, don't retry in case of error. Nothing Drake loads should have nasty
side-effects, so there should never be any reason to crash and resume.
Retrying might slow things down, which we definitely do NOT want.

Reasoning for not upstreaming this patch: Drake-specific option value.

--- mypy/moduleinspect.py
+++ mypy/moduleinspect.py
@@ -143,5 +143,5 @@
         if isinstance(res, str):
             # Error importing module
-            if self.counter > 0:
+            if False:
                 # Also try with a fresh process. Maybe one of the previous imports has
                 # corrupted some global state.
@@ -157,13 +157,15 @@
 
         Return the value read from the queue, or None if the process unexpectedly died.
         """
-        max_iter = 600
+        timeout_seconds = 720
+        timeout_increment = 0.05
+        max_iter = int(timeout_seconds / timeout_increment)
         n = 0
         while True:
             if n == max_iter:
                 raise RuntimeError("Timeout waiting for subprocess")
             try:
-                return self.results.get(timeout=0.05)
+                return self.results.get(timeout=timeout_increment)
             except queue.Empty:
                 if not self.proc.is_alive():
                     return None

--- mypy-1.19.1.dist-info/RECORD
+++ mypy-1.19.1.dist-info/RECORD
@@ -111,1 +111,1 @@
-mypy/moduleinspect.py,sha256=HCEI7yW61OkMNFqUqjuRB09HcTDpalcmtVBYjlWfxyo,6326
+mypy/moduleinspect.py,sha256=limKP3brX4IaLEam7mUYR6OsV1NP2F9MYGN-vnJQLQI,6428
